### DISCUSSION TITLE
whats Yggdrasil

### DISCUSSION THREAD

**Discussion by lemontetraline** [AntiCope/meteor-rejects#267] [bruh]
its minecraft auth system, but what is it doing in this addon and how to use it

**Comment by jklmm7xw** [comment]
yggdrasil is a tree

**Reply by lemontetraline** [reply]
wow that helped alot thanks youve opened my eyes on it tysm!!!

**Comment by GoofyBoi420** [comment]
Yggsraisil is an ALT managing thing for alts

**Reply by crosby-moe** [reply]
no

**Reply by GoofyBoi420** [reply]
It's a tree

**Reply by GoofyBoi420** [reply]
And I checked the read me ¯\_(ツ)_/¯
